File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.js
// Last updated: 10/03/2026, 10:03:01
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    let ans = [];
    
    for (let a = 0; a < nums.length; a++) {
        ans.push(nums[a]);
    }
    
    for (let a = 0; a < nums.length; a++) {
        ans.push(nums[a]);
    }

    
    return ans;
};

File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.cpp
// Last updated: 03/08/2026, 10:08:11
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans = nums;
        for(int num:nums){
            ans.push_back(num);
        }
        return ans;
    }
};